contracts/Splitter.sol

```diff
@@ -2,14 +2,21 @@ pragma solidity ^0.4.19;
 
 contract Splitter {
 
+    // Comment: If this variable is public we would not need to write a getBalance function -
+    // all public variables have autocreated get function see: http://solidity.readthedocs.io/en/develop/contracts.html#getter-functions
```

Comment:
Isn't the encapsulation a thing in solidity? Should be good to have some separation of the state and the behaviour of the classes(contracts);